### PR TITLE
[IMP]Migrated school module in v11

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: school/models/school.py [hunk 1]
```diff
@@ -235,14 +235,12 @@ def onchange_combine(self):
         self.name = str(self.standard_id.name
                         ) + '-' + str(self.division_id.name)
 
-    @api.multi
```
Comment (JayVora-SerpentCS) [inline school/models/school.py:239]: reason for removing multi ?

File: school/__manifest__.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
     'complexity': 'easy',
     'Summary': 'A Module For School Management',
     'images': ['static/description/EMS.jpg'],
-    'depends': ['hr', 'crm', 'account'],
+    'depends': ['hr', 'crm', 'account', 'web'],
```
Comment (JayVora-SerpentCS) [inline school/__manifest__.py]: no need to add web module, its auto install=true already.

**Comment by JayVora-SerpentCS** [comment]
Please improve the readme file too.

File: school/models/school.py (hunk 1 above)
Comment (Arpit-SerpentCS) [reply school/models/school.py:239]: We don't need decorator @api.multi here, as only @api.depends will do the job. I have referred base code and removed.

**Pull request merged**